semantic_model_generator/admin_app/pages/chat.py

```diff
@@ -0,0 +1,276 @@
+import json
+import os
+import time
+from typing import Any
+
+import pandas as pd
+import requests
+import sqlglot
+import streamlit as st
+from app_utils import (
+    _TMP_FILE_NAME,
+    SNOWFLAKE_ACCOUNT,
+    changed_from_last_validated_model,
+)
+from snowflake.connector import SnowflakeConnection
+
+from semantic_model_generator.data_processing.cte_utils import (
+    context_to_column_format,
+    expand_all_logical_tables_as_ctes,
+    logical_table_name,
+    remove_ltable_cte,
+)
+from semantic_model_generator.protos import semantic_model_pb2
+from semantic_model_generator.snowflake_utils.snowflake_connector import (
+    SnowflakeConnector,
+)
+
+HOST = os.environ["SNOWFLAKE_HOST"]
```

Comment:
should we have a util that validates env vars required for the admin app?

Reply:
nope, we should probably add one - gonna add to TODO